### ISSUE TITLE
[Abdul] Stage 3: Logistic Regression & Linear SVM (Baselines & Margin Classifiers)

### ISSUE THREAD

**Issue by dipk6545** [dipk6545/hotel-booking-aml#27]
### Assigned Models
1. **Logistic Regression** (Linear baseline)
2. **Linear SVM / SGDClassifier** (Margin-based classifier)

### Tasks
- [ ] Apply `StandardScaler` to numerical features.
- [ ] Train & evaluate baseline Logistic Regression and Linear SVM.
- [ ] Perform Hyperparameter Tuning:
  - `C`: [0.01, 0.1, 1.0, 10]
  - `penalty`: ['l1', 'l2']
  - `class_weight`: [None, 'balanced']
- [ ] Evaluate with 5-Fold Cross-Validation on the training set.
- [ ] Log test set metrics: ROC-AUC, F1-Score, Precision, Recall, Accuracy.
- [ ] Export your best-tuned model configuration for the team stacking ensemble.
